docs/projects/daq2/index.rst

```diff
@@ -0,0 +1,554 @@
+
+Given that the IP uses the same QUAD as the DAC, performing channel
+reconfiguration may affect the DAC and vice versa. When using the JESD204B
+framework, this is taken into consideration by software.
+
+To relax the constraints for PCB design, the n-th physical lane it is not
+connected to the n-th logical lane, therefore there is a remapping scheme
+between the physical and link layer to reorder the data streams.
+
+============ ======================= =======================
+DAC phy Lane FPGA Tx lane for Xilinx FPGA Tx lane for Altera
+============ ======================= =======================
+0            0                       0
+1            2                       3
+2            1                       1
+3            3                       2
+============ ======================= =======================
+
+Resources
+-------------------------------------------------------------------------------
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Product datasheets:
+
+  - :adi:`AD9680`
+  - :adi:`AD9144`
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :git-hdl:`DAQ2 HDL project source code <projects/daq2>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac`
+     - :ref:`axi_dmac`
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid`
+     - :ref:`axi_sysid`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom`
+     - :ref:`axi_sysid`
+   * - UTIL_CPACK2
+     - :git-hdl:`library/util_pack/util_cpack2`
+     - :ref:`util_cpack2`
+   * - UTIL_UPACK2
+     - :git-hdl:`library/util_pack/util_upack2`
+     - :ref:`util_upack2`
+   * - UTIL_ADXCVR for AMD
+     - :git-hdl:`library/xilinx/util_adxcvr`
+     - :ref:`util_adxcvr`
+   * - AXI_ADXCVR for Intel
+     - :git-hdl:`library/intel/axi_adxcvr`
+     - :ref:`axi_adxcvr intel`
+   * - AXI_ADXCVR for AMD
+     - :git-hdl:`library/xilinx/axi_adxcvr`
+     - :ref:`axi_adxcvr amd`
+   * - AXI_JESD204_RX
+     - :git-hdl:`library/jesd204/axi_jesd204_rx`
+     - :ref:`axi_jesd204_rx`
+   * - AXI_JESD204_TX
+     - :git-hdl:`library/jesd204/axi_jesd204_tx`
+     - :ref:`axi_jesd204_tx`
+   * - JESD204_TPL_ADC
+     - :git-hdl:`library/jesd204/ad_ip_jesd204_tpl_adc`
+     - :ref:`ad_ip_jesd204_tpl_adc`
+   * - JESD204_TPL_DAC
+     - :git-hdl:`library/jesd204/ad_ip_jesd204_tpl_dac`
+     - :ref:`ad_ip_jesd204_tpl_dac`
+   * - DATA_OFFLOAD
+     - :git-hdl:`library/data_offload`
+     - 	:ref:`data_offload`
+
+- :dokuwiki:`[Wiki] Generic JESD204B block designs <resources/fpga/docs/hdl/generic_jesd_bds>`
+- :ref:`jesd204`
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :git-linux:`DAQ2 Linux Device Tree <arch/arm64/boot/dts/xilinx/adi-daq2.dtsi>`
```

Comment:
The dtsi should be removed, and only the dts-es should be added:

- https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/kc705_fmcdaq2.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/vc707_fmcdaq2.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/microblaze/boot/dts/kcu105_fmcdaq2.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/arm/boot/dts/xilinx/zynq-zc706-adv7511-fmcdaq2.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev10-fmcdaq2.dts
- https://github.com/analogdevicesinc/linux/blob/main/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-rev10-fmcdaq2_m4_l4.dts